jnigen/lib/src/tools/build_summarizer.dart

```diff
@@ -29,11 +29,11 @@ Future<void> buildApiSummarizer() async {
   final pom = pkg.resolve('java/pom.xml');
   await Directory(toolPath).create(recursive: true);
   final mvnArgs = [
+    'package',
     '--batch-mode',
     '--update-snapshots',
     '-f',
-    pom.toFilePath(),
-    'assembly:assembly'
+    pom.toFilePath()
```

Comment:
It's a best practice to add trailing comma, after last element of any multi-line list.

Reply:
TIL! 

I've pushed a commit adding the trailing comma :smiley: